rails/spec/models/story_spec.rb

```diff
@@ -14,45 +14,51 @@
   describe 'validation' do
     let(:story_1) {build(:story)}
     let(:story_2) {create(:story, :with_speakers)}
-    it 'must have at least one speaker' do
-      expect(story_1).to_not be_valid
-      expect(story_2).to be_valid
+    describe 'must have at least one speaker' do
+      it {expect(story_1).to_not be_valid}
+      it {expect(story_2).to be_valid}
     end
   end
 
-  describe '.import_csv' do
-    it 'is tested against fixture file' do
-      expect(file_fixture('story_with_media.csv').read).not_to be_empty
+  context '.import_csv' do
+    describe 'is tested against fixture file' do
+      it {expect(file_fixture('story_with_media.csv').read).not_to be_empty}
     end
 
-    it 'delegates all attributes to decorator' do
-      fixture_data = file_fixture('story_with_media.csv').read
-      described_class.import_csv(fixture_data)
-
-      story = described_class.last
-      csv = CSV.parse(fixture_data, headers: true).first
-
-      expect(story.title).to eq csv[0]
-      expect(story.desc).to eq csv[1]
-      expect(story.speakers.map(&:name).join(',')).to eq csv[2]
-      expect(story.places.map(&:name).join(',')).to eq csv[3]
-      expect(story.interview_location.name).to eq csv[4]
-      expect(story.date_interviewed.strftime('%m/%d/%y')).to eq csv[5]
-      expect(story.interviewer.name).to eq csv[6].strip
-      expect(story.language).to eq csv[7]
-      expect(story.permission_level).to eq "anonymous"
-      expect(story.media.first.filename.to_s).to eq csv[8]
+    describe 'delegates all attributes to decorator' do
+      before(:all) do
+        @fixture_data = file_fixture('story_with_media.csv').read
+        described_class.import_csv(@fixture_data)
+      end
+      let!(:story) {described_class.last}
+      # puts story.title
```

Comment:
@slaloggia whats going on here? are these comments just need to be removed?

Reply:
They are indeed. Will fix.

Reply:
Should be good to go. Double-checked all the spec files for other stray comments and removed a couple in other places.